### DISCUSSION TITLE
Loading ONNX model with webui

### DISCUSSION THREAD

**Discussion by forpr1093** [oobabooga/text-generation-webui#5448] [Q&A]
Hi, I am trying to load this ONNX model: https://huggingface.co/neuralmagic/Nous-Hermes-2-SOLAR-10.7B-pruned50-quant-ds/tree/main. 
But I couldn't make it work using any model loader...

Any help will be appreciated!

**Comment by buildmine10** [comment]
As far as I can tell text-generation-webui doesn't support ONNX. I cannot even download the models via the gui. I'll report back once it finishes downloading directly.
